### PR TITLE
Keeper integration

### PR DESCRIPTION
**Pull request by idimov-keeper**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Initial commit for a new integration - Keeper Secrets Manager
Related docker image in [PR#8448](https://github.com/demisto/dockerfiles/pull/8448)

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
@JasBeilin - Please review and approve https://github.com/demisto/dockerfiles/pull/8448 before reviewing this one.

File: Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    - contextPath: KeeperSecretsManager.Records.type
+      description: Record Type.
+      type: String
+    - contextPath: KeeperSecretsManager.Records.title
+      description: Record Title.
+      type: String
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/keeper-ksm:latest
```
Comment (kgal-pan) [inline Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.yml:193]:
```suggestion
  dockerimage: demisto/keeper-ksm:1.0.0.33394
```

**Comment by kgal-pan** [comment]
https://github.com/demisto/content/pull/20875 blocking [UT build](https://app.circleci.com/pipelines/github/demisto/content/186914/workflows/e8711195-bb48-4f7f-9718-63b06012c848/jobs/561304?invite=true#step-108-19)

**Comment by idovandijk** [comment]
If any playbooks are added to this PR please reassign me to review, for now I'm unassigning :)

**Comment by kgal-pan** [comment]
> #20875 blocking [UT build](https://app.circleci.com/pipelines/github/demisto/content/186914/workflows/e8711195-bb48-4f7f-9718-63b06012c848/jobs/561304?invite=true#step-108-19)

@idimov-keeper - Please sync with `master`.

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Thank you for this contribution!
Please see my comment below.

File: Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.py [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+        message = 'ok'
+    except Exception as e:
+        message = str(e)
+    return message
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
```
Comment (kgal-pan) [inline Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.py:604]: Please add to increase coverage and [overcome build failure](https://app.circleci.com/pipelines/github/demisto/content/191887/workflows/2a73ea46-3107-4b7f-8f54-231d68186492/jobs/570866/steps?invite=true#step-108-14).

```suggestion
def main() -> None:   # pragma: no cover
```

File: Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.py [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    credentials = demisto.params()['credentials']
```
Comment (kgal-pan) [inline Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.py:611]:
```suggestion
    credentials = demisto.params().get('credentials')
```

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**